nix/verilator.nix

```diff
@@ -56,12 +56,14 @@ with pkgs; clangStdenv.mkDerivation rec {
   preConfigure = "autoconf";
 
   postPatch = ''
-    patchShebangs bin/* src/{flexfix,vlcovgen} test_regress/{driver.pl,t/*.pl}
+    patchShebangs bin/* src/{flexfix,vlcovgen}
   '';
 
   postInstall = lib.optionalString stdenv.isLinux ''
     for x in $(ls $out/bin/verilator*); do
       wrapProgram "$x" --set LOCALE_ARCHIVE "${glibcLocales}/lib/locale/locale-archive"
     done
   '';
+
+  doCheck = false;
```

Comment:
If I understood the documentation correctly, this disables checkPhase. Why are you disabling it ?

Reply:
We do so for all underlying tools (except for the Yosys plugins which have a much higher risk of disintegrating.) Building and running tests takes time.